docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -335,6 +335,39 @@ Payload objects with the following scopes take the form that follows:
 }
 ```
 
+## Delivery exception hooks
+
+| Name / Scope             | Description |
+|:-------------------------|:------------|
+| store/hook/deliveryException | Once configured by the API client, this hook fires when the platform has issues delivering regular hooks. The system will attempt to send this new hook and provide information regarding unsuccessful webhook delivery. There can only be one hook per store_id or clientId. The destination must be different from all other hook destinations.  |
+
+There are three different error codes available:
+* 90001 - The hook couldn’t be delivered, but system will retry. This type is limited to be sent only once in 10 minutes per destination.
+* 90002 - The hook couldn’t be delivered. All attempts are exhausted. The hook is going to be disabled.
+* 90003 - The hook couldn’t be delivered because domain was blocked.
```

Comment:
```suggestion
* 90003 - The hook couldn’t be delivered because domain was temporarily blocked due to many unsuccessful attempts to deliver a hook.
```

Reply:
btw @kskomro @bc-tayvs are we blocking the destination domain or destination full URL?

Reply:
we are blocking entire domain: https://github.com/bigcommerce/webhooks-manager/blob/master/core-common/src/main/scala/com/bigcommerce/core/redis/WebhooksRedisOps.scala#L32C78-L32C82

Reply:
Great!  Thank you @kskomro